css/css-text/text-indent/text-indent-with-absolute-pos-child.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+<meta charset="utf-8">
+<title>This tests that first line text-indent is applied properly when the child is a statically positioned out-of-flow renderer.</title>
+<link rel="author" title="Zalan Bujtas" href="mailto:zalan@apple.com">
```

Comment:
`Alan Bujtas` (also please ask Alan if he wants to be set as author). cc @alanbujtas

Reply:
I asked Alan and he said no, so I'll remove it.